.github/workflows/cherry-pick-next-to-master.yml

```
@@ -0,0 +1,29 @@
+on:
+  pull_request:
+    branches:
+      - next
+    types: ['closed']
+
+jobs:
+  cherry_pick_to_master:
+    runs-on: ubuntu-latest
+    name: Cherry pick into master
+    permissions:
+      pull-requests: write
+      contents: write
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'needs cherry-pick') && github.event.pull_request.merged == true }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cherry pick and create the new PR
+        uses: carloscastrojumo/github-cherry-pick-action@v1.0.9
+        with:
+          branch: master
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: 'Cherry-pick of #{old_pull_request_id}'
```

```suggestion
          inherit_labels: false
          body: 'Cherry-pick of #{old_pull_request_id}'
```

Restoring what was removed in https://github.com/mui/mui-x/pull/11382/commits/80d7eb5fb7ce96c8cca47be62c5392642ab26f58

---

Same

---

`inherit_labels: true` is the default behavior.
It's a default `true` boolean prop. 😄

---

I'll merge this one in that case. Thanks for the useful feedback anyways. 👍